tests/Exception/AwsExceptionTest.php

```diff
@@ -55,6 +55,13 @@ public function testProvidesResponse()
         $this->assertSame($response, $e->getResponse());
     }
 
+    public function testProvidesErrorMessage()
+    {
+        $command = new Command('foo');
+        $e = new AwsException('Foo', $command, ['message' => "test error message"]);
+        $this->assertSame("test error message", $e->getErrorMessage());
```

Comment:
Might be overkill, but what you're explicitly fixing is that the concise message is not the same as the `$message` argument passed to the `AwsException` constructor ('Foo' in this case); you could add an assertion that the value returned by the new method is not `'Foo'`.